android/settings.gradle

```diff
@@ -18,3 +18,5 @@ include ':react-native-status'
 project(':react-native-status').projectDir = new File(rootProject.projectDir, '../modules/react-native-status/android')
 include ':react-native-status-keycard'
 project(':react-native-status-keycard').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-status-keycard/android')
+include ':react-native-blur'
+project(':react-native-blur').projectDir = new File(rootProject.projectDir, '../node_modules/@react-native-community/blur/android')
```

Comment:
Project name don't allow `/` `@`, So instead of using `@react-native-community/blur` as project name, library name is being used. ([ref](https://github.com/Kureev/react-native-blur/issues/213#issuecomment-311414060))